dicthash/dicthash.py

```diff
@@ -49,58 +45,66 @@ def _save_convert_float_to_int(x):
     return int(x * FLOAT_FACTOR)
 
 
-def _unpack_value(value, prefix='', whitelist=None, blacklist=None):
+def _unpack_value(value: Union[dict, Iterable, float, int],
+                  prefix: str ='', whitelist: Optional[List[Hashable]] =None,
```

Comment:
```suggestion
                  prefix: str='', whitelist: Optional[List[Hashable]]=None,
```

Reply:
Style guide says to have whitespace on each sides of the `=`: https://www.python.org/dev/peps/pep-0008/#id53
So I changed in this direction.

Reply:
wow, ok, that's _really_ unintuitive. do you happen to know the reason for this? i think the type annotation shouldn't change the style convention for default arguments!